docs/upgrading.md

```diff
@@ -5,6 +5,16 @@ Below we describe the additional manual steps required to upgrade between differ
 
 # Open Autonomy
 
+## `v0.8.0` to `v0.9.0`
+
+Breaking changes
+
+- the `max_participants` have been removed. 
+- the `consensus_threshold` should now be specified in the `setup` parameters. If `null`, then it is calculated automatically.
```

Comment:
I would cluster this and the above as subpoints under changes to the configuration